### PR TITLE
Top-Level Community Item with better information

### PR DESCRIPTION
**Pull request by nickik**
Hi,

I listened to the changelog podcast and I had the same confusing experiance with the documentation and the website.

I took the suggestions from the podcast and implemented them.

I have only done the minimal changes without updates for the languages and such.

If this is approved I would do the changes in layout and such to the languages as well. I don't know where the translatins came from.

Changes:

- Added 'Community' Tab explaining the communication channels
- Remove Twitter and Forum as top level navigation links
- Removed duplicated information on 'Documentation'

If this goes threw I would also suggest updating the 'Contributers.md' and fix it with a link to the Community page in order not to dublicate that information.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title = "Community"
++++
+
+This page attempts to explain how the Redox OS community is organised and help you navigate it.
+
+
+<a id="chat"></a>
+## [Chat](https://chat.redox-os.org)
+
+The quickest and most open way to communicate with the Redox team is on our chat server. Currently, you can only get an invite by sending an email request to info@redox-os.org, which might take a little while, since it's not automated. Simply say you'd like to join the chat. We're working on an better way to do this, but this is the best way right now.:wq
```
Comment (anxiousmodernman) [inline community.md]: trailing ":wq" here

**Comment by jackpot51** [comment]
To do this correctly, the community.md file needs to be community.en.md, and an entry needs to be added for every language.

**Comment by jackpot51** [comment]
You do not have to translate for every language, having a copy of the english entry is fine as a placeholder

**Comment by nickik** [comment]
Thank you for the feedback. I will do it with placeholders and update the pull request.

**Comment by jackpot51** [comment]
I appreciate that @nickik !

**Comment by nickik** [comment]
Did the updates. Without translation. I will open an issue for translations.

File: config.toml [hunk 2]
```diff
@@ -48,32 +44,36 @@ languageCode = "zh-cn"
 weight = 1
 name = "文档"
 url = "/docs/"
-[[languages.zh.menu.main]]
+[[languages.en.menu.main]]
```
Comment (jackpot51) [inline config.toml]: This should not have changed from .zh

File: config.toml [hunk 3]
```diff
@@ -48,32 +44,36 @@ languageCode = "zh-cn"
...
 name = "论坛"
 url = "https://discourse.redox-os.org/"
 [[languages.zh.menu.main]]
-weight = 6
+weight = 7
 name = "代码库"
 url = "https://github.com/redox-os/redox"
 [[languages.zh.menu.main]]
-weight = 7
+weight = 8
```
Comment (jackpot51) [inline config.toml]: Forum and twitter links can be removed

Comment (nickik) [reply config.toml]: Somehow I missed 'zh', I have updated.

**Pull request merged**